integration/struct/google/protobuf/struct.ts

```diff
@@ -466,3 +462,11 @@ if (util.Long !== Long) {
   util.Long = Long as any;
   configure();
 }
+
+function isObject(value: any): boolean {
+  return typeof value === 'object' && value !== null;
+}
```

Comment:
👍 

Just in case type narrowing matters at some point, have a look at this implementation I wrote for a different project: https://github.com/cosmos/cosmjs/blob/v0.27.0-rc2/packages/utils/src/typechecks.ts#L1-L12